test/integration-test-wrapper.py

```diff
@@ -34,6 +35,55 @@
 """
 
 
+def dump_coredumps(args: argparse.Namespace, journal_file: Path) -> bool:
+    # Collect executable paths of all coredumps and filter out the expected ones.
+    # The following are excluded:
+    #   sleep/bash - intentional SIGABRT caused by TEST-57
+    #   systemd-notify - intermittent (and intentional) SIGABRT caused by TEST-59
+    #   test-execute - intentional coredump in TEST-02
+    #   test(-usr)?-dump - intentional coredumps from systemd-coredump tests in TEST-74
+    exclude_regex = re.compile("/(bash|sleep|systemd-notify|test-execute|test(-usr)?-dump)")
```

Comment:
Maybe, it is better to allow to append or override this by each test.
See https://github.com/systemd/systemd-centos-ci/blob/86adb3039c359214eccafa933e71ca001723143f/common/utils.sh#L10-L21 .

Reply:
@yuwata I would see if there is a need first, we'll find out as soon as the integration tests finish